hack/e2e-test.sh

```diff
@@ -106,7 +106,7 @@ while [ $i -le ${max_tries} ]; do
          AWS_SECRET_ACCESS_KEY="${AWS_SECRET_ACCESS_KEY}" \
          BASE_DOMAIN="${BASE_DOMAIN}" \
          HIVE_IMAGE="${HIVE_IMAGE}" \
-         CUSTOM_RELEASE_IMAGE="${RELEASE_IMAGE}" \
+         RELEASE_IMAGE="${RELEASE_IMAGE}" \
          CLUSTER_IMAGE_SET="custom" \
```

Comment:
@dgoodwin this will cause the e2e to fail because we're pointing to an imageset that doesn't exist. Because we're no longer creating an imageset in the clusterdeployment template, maybe we should add another template for an imageset and have the e2e create one of those for its release image and then point the clusterdeployment to that.

Reply:
Is the operator in play for e2e? I thought it would come up, create the default image sets, one of which is the one referenced by default in the cluster deployment template. Then by setting an explicit RELEASE_IMAGE we would be overriding that.

Reply:
Ok update with a new template for custom image set, it removes the ability to set per CD overrides. It uses the same vars, HIVE_IMAGE and RELEASE_IMAGE, and includes a custom image set that uses those vars. INSTALLER_IMAGE is removed.